conmon-rs/common/proto/conmon.capnp

```diff
@@ -38,9 +38,8 @@ interface Conmon {
 
     struct ExecSyncContainerResponse {
         exitCode @0 :Int32;
-        stdout @1 :Text;
-        stderr @2 :Text;
+        stdio @1 :Data;
```

Comment:
I think we still need a differentiation between the two, even if a tty request only sends in stdout

Reply:
Good, added it back.